rspace/src/main/scala/coop/rchain/rspace/nextgenrspace/history/History.scala

```diff
@@ -40,6 +40,9 @@ object History {
     Skip(ByteVector(affix), NodePointer(pb.hash))
 
   type KeyPath = Seq[Byte]
+
+  val keyPathOrdering: Ordering[KeyPath] =
```

Comment:
Does this need to be defined explicitly? Isn't importing `Ordering.Implicits` wherever such ordering is needed enough?

Reply:
the Seq implicit in hidden AFAICT

Reply:
from documentation
```
    /** Not in the standard scope due to the potential for divergence:
      * For instance `implicitly[Ordering[Any]]` diverges in its presence.
      */
    implicit def seqDerivedOrdering[
```